models/post.go

```diff
@@ -74,32 +72,17 @@ func (p *Post) Share() (err error) {
   return
 }
 
-func GetPostsNearLocation(lat string, lon string, userID uint, radius string, numResults int) (posts []Post, err error) {
+// TODO: Should refactor to use postGIS types but GORM doesn't support them, so that's a larger discussion
```

Comment:
https://github.com/dewski/spatial ?

Reply:
Yeah I saw that when I was googling around for this. idk maybe